### ISSUE TITLE
Code from Intro to NLP course causing bus error (crashing python) on OSX on Apple hardware

### ISSUE THREAD

**Issue by bshawwi** [huggingface/transformers#36007]
### System Info

transformers       4.48.2
OSX Sequoia 15.3
python 3.12.5

NLP Course, Chapter 2, Handling multiple sequences
The following block of code causes the bus error at the next-to-last (ie penultimate) line of output = model(input_ids)

import torch
from transformers import AutoTokenizer, AutoModelForSequenceClassification

checkpoint = "distilbert-base-uncased-finetuned-sst-2-english"
tokenizer = AutoTokenizer.from_pretrained(checkpoint)
model = AutoModelForSequenceClassification.from_pretrained(checkpoint)

sequence = "I've been waiting for a HuggingFace course my whole life."

tokens = tokenizer.tokenize(sequence)
ids = tokenizer.convert_tokens_to_ids(tokens)

input_ids = torch.tensor([ids])
print("Input IDs:", input_ids)

output = model(input_ids)
print("Logits:", output.logits)

### Information

- [x] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

1. Setup per the Chapter 0 of the NLP course
2. run command: source .env/bin/activate
3. run command: python3
4. copy and paste the code identified above

python3 crashes

### Expected behavior

Expected behavior is, as per the tutorial, the following output:

Input IDs: [[ 1045,  1005,  2310,  2042,  3403,  2005,  1037, 17662, 12172,  2607, 2026,  2878,  2166,  1012]]
Logits: [[-2.7276,  2.8789]]

**Comment by Rocketknight1** [comment]
That code looks quite straightforward and runs fine for me, so I suspect the crash is an environment issue with your torch setup! I'm afraid we can't debug local environment issues because they're impossible for us to reproduce, and they're not bugs in `transformers` itself.
